benchmark/local.rb

```diff
@@ -111,6 +112,28 @@ def commenter_names
   end
 end
 
+# --- Primalize serializers ---
```

Comment:
Could you move that down, just above the line:

`# --- Representable serializers ---`

to keep this alphabetically sorted as well?